docs/data-collection/notes-scanning.md

```diff
@@ -162,6 +162,8 @@ Adjust settings by pressing the respective button and then turning the central k
 
 !!! warning "The central knob (button 1) will turn off the alarm if pushed when the alarm is on"
 
+!!! warning "On {{settings.mri.models.mri2}} and {{settings.mri.models.mri3}}, the alarm can be turned off by touching the tactil screen on the bore of the scanner"
```

Comment:
Okay, if that is the case, then we need to merge the two warnings and highlight that the procedure is different depending on the system:

```suggestion
!!! warning "Turning off the alarm"

    The alarm is turned off differently depending on the system:

       * {{settings.mri.models.mri1}}: push the central knob (button 1).
       * {{settings.mri.models.mri2}} and {{settings.mri.models.mri3}}: touch one of the tactile screens on the scanner's front.
```